com.github.alainm23.byte.json

```diff
@@ -0,0 +1,65 @@
+{
+    "app-id": "com.github.alainm23.byte",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.alainm23.byte",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--metadata=X-DConf=migrate-path=/com/github/alainm23/byte/",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+        "--talk-name=org.mpris.MediaPlayer2",
+        "--own-name=org.mpris.MediaPlayer2.com.github.alainm23.byte"
```

Comment:
You own this name by default.

Reply:
```suggestion
```
You own that name by default with Flatpak.

Reply:
@alainm23 ^^^

Reply:
I'm sorry, What do I need to do here?

Reply:
You need to remove this line. You already own that bus name, no need to request it here again.